File: Robert/Programmers/Level1/이상한문자만들기.py
# https://programmers.co.kr/learn/courses/30/lessons/12930


def solution(s):
    result = []
    for string in s.split(" "):
        _string = ""
        for index in range(len(string)):
            _string += string[index].lower() if (index + 1) % 2 == 0 else string[index].upper()
        result.append(_string)
    return ' '.join(result)


s = "try hello world"
print(solution(s))

File: Robert/Programmers/Level1/이상한문자만들기.ts
function solution(s) {
  const string_list = s.split(" ");
  const result = [];
  string_list.forEach((element) => {
    let _string = "";
    for (let i = 0; i < string_list.length; i++) {
      _string += element[i].toLowerCase()
        ? (i + 1) % 2 == 0
        : element[i].toUpperCase();
    }
    result.push(_string);
  });

  return result;
}

const s = "try hello world";
console.log(solution(s));
